nautobot/core/templates/utilities/render_jinja2.html

```diff
@@ -10,7 +10,7 @@
     }
     #render_jinja_template_form textarea {
```

Comment:
Is a `<textarea>` the right widget to use for the rendered config on the right or should it just be something like a `<pre>` block?

Reply:
I switched to a `<pre><code>` block for this functionality in golden config. They both behave similarly for this use case but I had to override some bootstrap default css styles for the `<pre><code>` element to work, including the `white-space` style because bootstrap defaults it to `pre-wrap`. Now that I've done a little more testing I may switch this back to a textarea in GC because a `<pre><code>` block doesn't allow you to do a "select all" like you can in a textarea.